app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show destroy]
+
+  def index
+    @posts = current_user.feed.paginate(page: params[:posts_page], per_page: 10)
```

Comment:
We do have the exact same line in `users_controller`  `index` action. How different is this `@posts` from the one in `users_controller` index?.

I mean are we showing `posts` independent of the users also? Or do we only intend to show posts in either the user's timeline or feed? If so, we can probably have both `indexes` of posts under the tutelage of the `user` as either a timeline post or a feed post (I believe you have a method for feeds).